source/reference/minio-mc/mc-ilm-tier-add.rst

```diff
@@ -225,6 +226,8 @@ The command accepts the following arguments:
 
    The bucket on the remote tier to which MinIO transitions objects.
 
+   For ``azure`` remote tiers, this value corresponds to the :azure-docs:`Container name <storage/blobs/storage-blobs-introduction#containers>`
```

Comment:
Not sure `Container name` needs to be capitalized here.

Reply:
It's an actual "thing" in Azure - so 

Storage Account -> Containers -> Blobs

so it's distinct from the common "container" being a VM :grimacing: